### PR TITLE
wang_1997_lateral_1

### PR DESCRIPTION
**Pull request by markus-chem**
add lit folder

### CODE REVIEW

File: literature/wang_1997_lateral_1/wang_1997_lateral_1.bib [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@techreport{wang_1997_lateral_1,
+  title={Lateral interaction energy derived from Frumkin isotherm for c (2 x 2) Br/Ag (100)},
```
Comment (DunklesArchipel) [inline literature/wang_1997_lateral_1/wang_1997_lateral_1.bib:2]:
```suggestion
  title={Lateral interaction energy derived from Frumkin isotherm for c(2x2) Br/Ag(100)},
```
```suggestion
  title={Lateral interaction energy derived from Frumkin isotherm for c (2 x 2) Br/Ag (100)},
```

File: literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# This file describes a figure from a publication containing electrochemical data.
+# For the `echemdb/website` project, the source, curator and electrochemical system is mandatory.
+# Figure description is optional. It is created automatically by the cv module of svgdigitizer
+
+source:
+    version: 1
+    doi:  #only the number without preceeding doix or https:
```
Comment (DunklesArchipel) [inline literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml:7]: please add DOI

**Review by DunklesArchipel — CHANGES_REQUESTED** [review]
A DOI is missing.

File: literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml (hunk 2 above)
Comment (markus-chem) [reply literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml:7]: The publication seems to have no DOI. See: https://www.osti.gov/biblio/537264

Comment (DunklesArchipel) [reply literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml:7]: Then I guess we should use in this case: www.osti.gov/biblio/537264

Comment (DunklesArchipel) [reply literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml:7]:
```suggestion
    doi: www.osti.gov/biblio/537264 #only the number without preceeding doix or https:
```

**Comment by markus-chem** [comment]
> A DOI is missing.

I have committed your suggested change with the link instead of the DOI.

**Comment by markus-chem** [comment]
So, we can merge now? I can't, because there are still changes requested.

**Review by DunklesArchipel — COMMENTED** [review]
can we leave this open for the moment until we merge #77.
With that PR this one will result in a failed test, since the URL on the entry page will be constructed from the DOI. 
Probably we need to include a new field called `url`. So, if DOI is not available, we would use the URL.

**Comment by DunklesArchipel** [comment]
Merge when #84 is done.

File: literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
+# This file describes a figure from a publication containing electrochemical data.
+# For the `echemdb/website` project, the source, curator and electrochemical system is mandatory.
+# Figure description is optional. It is created automatically by the cv module of svgdigitizer
+
+source:
+    version: 1
+    doi: www.osti.gov/biblio/537264 #only the number without preceeding doix or https:
```
Comment (DunklesArchipel) [inline literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml:7]:
```suggestion
    url: www.osti.gov/biblio/537264 #only the number without preceeding doix or https:

```

Comment (DunklesArchipel) [reply literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml:7]: Will be supported in #77 .

File: literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                  value: 
+                  unit:
+            - name: KClO4
+              type: salt
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value: 40
+                  unit: mM #[M, mM, µM, g kg-1, ...]
```
Comment (DunklesArchipel) [inline literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml:69]:
```suggestion
                  unit: mmol / l #[M, mM, µM, g kg-1, ...] 

```

File: literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: KBr # can be trivia name, sum formula, etc
+              concentration:
+                  value: 10
+                  unit: mM #[M, mM, µM, g kg-1, ...]
```
Comment (DunklesArchipel) [inline literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml:41]:
```suggestion
                  unit: mmol / l #[M, mM, µM, g kg-1, ...]

```

File: literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.svg [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    <text
+       xml:space="preserve"
+       style="font-size:85.3333px;line-height:1.25;font-family:sans-serif"
+       x="366.06207"
+       y="1839.408"
+       id="text4334"><tspan
+         sodipodi:role="line"
+         id="tspan4332"
+         x="366.06207"
+         y="1839.408">y1: -15 uA/cm2</tspan></text>
```
Comment (DunklesArchipel) [inline literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.svg:74]:
```suggestion
         y="1839.408">y1: -15 uA / cm2</tspan></text>
```

File: literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.svg [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+    <text
+       xml:space="preserve"
+       style="font-size:85.3333px;line-height:1.25;font-family:sans-serif"
+       x="750.2254"
+       y="880.37329"
+       id="text2380"><tspan
+         sodipodi:role="line"
+         id="tspan2378"
+         x="750.2254"
+         y="880.37329">y2: 15 uA/cm2</tspan></text>
```
Comment (DunklesArchipel) [inline literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.svg:57]:
```suggestion
         y="880.37329">y2: 15 uA / cm2</tspan></text>
```

**Review by DunklesArchipel — APPROVED** [review]

**Pull request merged**